src/renderer/mod.rs

```diff
@@ -336,7 +339,7 @@ pub struct Renderer {
     quad: SurfaceSharedData,
     frame_size: (u32, u32),
     ambient_color: Color,
-    quality_settings: QualitySettings,
+    pub quality_settings: QualitySettings,
```

Comment:
This should be private, check `set_quality_settings` of Renderer, it doing a lot more than just setting a field. If it would be exposed as public then setting new value to it wouldn't have effect.

Reply:
Ah okay. I didn't know that method existed.

Reply:
@mrDIMAS resolved. PR ready to be merged